src/pages/CustomerRequestOverduePayment/validateEmails.ts

```diff
@@ -0,0 +1,22 @@
+import { string } from 'yup'
+
+// Matches a comma surrounded by optional whitespace
+const SEPARATOR_REGEX = /\s*,\s*/
```

Comment:
Is there a specific case that requires this regex?

At first I thought performing a `split(',').map(e => e.trim())` would be enough but maybe I was wrong!
And as this would needs an extra filter to check for the email presence we could replace all of it by a reduce, that will trim and push value only of present.

Anyway, don't need to change this logic if it works! Just curiosity

Reply:
I've totally overcomplicated my function. This solution works well haha. Updating it!